build.gradle

```diff
@@ -28,6 +27,7 @@ subprojects {
 
 
 	apply plugin:'java'
+	apply plugin:'groovy'
 //	apply plugin: 'com.github.johnrengelman.shadow'
```

Comment:
Do you need this plugin?

Reply:
I think so, there are groovy tests. IntelliJ complains without it. Are you seeing any errors (in which IDE)?

Reply:
Not sure it is configured correctly to remove the IntelliJ errors however